src/content/docs/en/getting-started.mdx

```diff
@@ -12,9 +12,9 @@ hero:
     - text: Install Astro
       icon: rocket
       link: /en/install-and-setup/
-      variant: primary
     - text: Learn about Astro’s features
       icon: right-arrow
+      variant: minimal
```

Comment:
Perhaps we can make this change across all languages and mark this PR as `i18nIgnore`? What do you think? Otherwise some languages will have a double “primary” button until they are updated, like this:
<img width="447" alt="image" src="https://github.com/user-attachments/assets/b8d1738f-05e6-415d-aa91-602e087783f9">

Reply:
Ok. I will change all other languages and mark this PR as `i18nIgnore`